CMSgov/mdct-mcr
# add hydration mechanism to formJson parsing

## Description

To display fields that have been pre-filled with data (whether disabled or not) while maintaining the static nature of the form json, we need a hydration mechanism to fill the field values with dynamic data.

Changes:
- Added hydration utility function to handle hydrating form fields.
- Added temporary fake data to `McparReportPage` for state and program name. These are both things we can get from the database once we have program creation figured out.
- Modified existing forms to conform to modified types and add in necessary hydration data.
- Tweaked some related styles and types.

### How to test

1. Navigate to `/mcpar/program-information/point-of-contact`. Observe that field A1 has been hydrated with dynamic data.
2. Complete form with valid fill and continue. Observe that field A6 has been hydrated with dynamic data.
3. All tests pass.

### Changed Dependencies

n/a

## Code author checklist

## Reviewer checklist (two different people)
